### PR TITLE
update max Collators for MB Alpha and update release version tag

### PR DESCRIPTION
**Pull request by themacexpert**
- Update max Collators for Moonbase Alpha to 60.
- Update version release tag to 0.14.2

### CODE REVIEW

File: variables.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ networks:
     hex_chain_id: '0x507'
     chain_spec: alphanet
     block_explorer: https://moonbase-blockscout.testnet.moonbeam.network/
-    parachain_release_tag: v0.13.2 # must be in this exact format for links to work
+    parachain_release_tag: v0.14.2 # must be in this exact format for links to work
     parachain_sha256sum: 9e748a0c91d5ed5f999bae91ceed1343db0d9dda40270201abf00db067c86a71
```
Comment (albertov19) [inline variables.yml:16]: Thanks for update the version, but you need to also update the `sha256_sum` of the new image.

**Review by albertov19 — CHANGES_REQUESTED** [review]
Added a comment. You also need to update the release tag for Moonriver I guess, and the `sha_sum`

Thanks!

File: variables.yml (hunk 1 above)
Comment (themacexpert) [reply variables.yml]: Thank you @albertov19! Fixed

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**